content/docs/goFSH/running/_index.md

```diff
@@ -40,16 +50,10 @@ GoFSH does not require any configuration, but if the input FHIR artifacts depend
 GoFSH can still generate FSH when dependencies are omitted, but the resulting FSH will be incomplete.
 {{% /alert %}}
 
-### Limitations on Inputs
-
-Note that GoFSH is not complete, and cannot be applied to all FHIR artifacts. The capabilities and limitations of GoFSH are given below.
+{{% alert title="Warning" color="warning" %}}
+GoFSH cannot generate FSH from XML inputs, only JSON is allowed.
+{{% /alert %}}
```

Comment:
This of course will have to be changed if we add support for XML.

Reply:
I'll take it out and put the pressure on myself